modules/rosa-upgrading-cli-tutorial.adoc

```diff
@@ -58,9 +57,51 @@ $ rosa upgrade cluster --cluster=<cluster_name|cluster_id> --control-plane
 ----
 endif::rosa-hcp[]
 +
-The cluster is scheduled for an immediate upgrade. This action can take an hour or longer, depending on your workload configuration, such as pod disruption budgets.
+[NOTE]
+====
+If you are upgrading an AWS Security Token Service (STS) cluster, this command starts an interactive IAM Roles/policies upgrade mode process that verifies the account and operator role policies for the chosen cluster are compatible with the target version of the upgrade. If the policies are not compatible with the chosen upgrade version, the CLI prompts you to upgrade them.
```

Comment:
" If the policies are not compatible with the chosen upgrade version, the CLI prompts you to upgrade them. "
Is this correct? Does CLI prompts users to upgrade them?

Reply:
What about UI? This paragraph is under the heading " Upgrading a ROSA Cluster" which I suppose is "generic" because there are topics separately for "Upgrading through ROSA CLI" and "Upgrading through OCM Console" that would be specific to CLI and console respectively?

Reply:
@ravitri can you comment on @bchandra-ocp questions: "Is this correct? Does CLI prompts users to upgrade them?" for end of line 62.

Reply:
@bchandra-ocp again I am confused as I was informed that this is for Classic only and like you said we have 2 separate modules/topics for CLI and Console - so does the OCM console not cover the UI? thanks

Reply:
> " If the policies are not compatible with the chosen upgrade version, the CLI prompts you to upgrade them. " Is this correct? Does CLI prompts users to upgrade them?

Changed this to "automatically upgrades"

Reply:
> @bchandra-ocp again I am confused as I was informed that this is for Classic only and like you said we have 2 separate modules/topics for CLI and Console - so does the OCM console not cover the UI? thanks

@bchandra-ocp are you ok with this now, if you look at full preview:
https://76632--ocpdocs-pr.netlify.app/openshift-rosa/latest/upgrading/rosa-upgrading-sts

Reply:
@jameszwang so change from 
"
If you are upgrading an AWS Security Token Service (STS) cluster, this command starts an interactive IAM Roles/policies upgrade mode process that verifies the account and operator role policies for the chosen cluster are compatible with the target version of the upgrade. If the policies are not compatible with the chosen upgrade version, the CLI automatically upgrades them.
"
to?
do you want to reword for me so we get this right this time?

Reply:
@fmcdonal I think it would be better like this:
“
If you are upgrading an AWS Security Token Service (STS) cluster, this command starts an interactive IAM Roles/policies upgrade mode process that verifies the account and operator role policies for the chosen cluster are compatible with the target version of the upgrade. If the policies are not compatible with the chosen upgrade version, the CLI automatically upgrades them in the auto mode.
“

Reply:
added "in auto mode", thanks for clarifying @jameszwang